app/routes/docs/styleguide.md

```diff
@@ -112,13 +110,9 @@ Acceptable content for the GCN Circulars falls within only four areas:
 - Do not produce excessively long or unfocused Circulars
 - Do not submit automated circulars from templates without clear language and grammar
 - Do not imply you alone discover a transient that has already been reported
-- Do not neglect to mention potential systematic errors if they are expeced
+- Do not neglect to mention potential systematic errors if they are expected
 - Do not declare an event start time if you likely missed the beginning
 
 ## Do's
 
 - Work together to discover secrets of the Universe
-
-### Acknowledgements
-
-The GCN Circular service has benefited from numerous discussions and beta-testing with many people, most notably Roland Vanderspek, Ralph Wijers, James Rhoads, and Hye-Sook Park, plus the people at the 4th Huntsville GRB Workshop evening SIG session and the Woods Hole GRB Workshop. The overwhelming development work was performed by Scott Barthelmy. This Style Guide was adapted from the original with input from Kevin Hurley.
```

Comment:
Why remove the acknowledgements? It is probably true that these people established norms that have held for decades.

Reply:
The style guide isn't the right place for acknowledgements.  It should go into the About page, or history.